Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/adapters/GameAdapter.java

```diff
@@ -223,16 +225,12 @@ public boolean onLongClick(View view)
 
 		// Get the ID of the game we want to look at.
 		// TODO This should be all we need to pass in, eventually.
-		// String gameId = (String) holder.gameId;
+		String gameId = (String) holder.gameId;
 
 		FragmentActivity activity = (FragmentActivity) view.getContext();
-		GameDetailsDialog.newInstance(holder.title,
-				holder.description,
-				holder.country,
-				holder.company,
-				holder.path,
-				holder.screenshotPath).show(activity.getSupportFragmentManager(), "game_details");
-
+		Intent intent = new Intent(activity, GameDetailsActivity.class);
```

Comment:
Can we move this into a launch() method inside GameDetailsActivity to be consistent with the rest of the project?